benchmark/bm_write_queue_collector_thread.py

```diff
@@ -39,6 +39,7 @@ def main():
     thr_coll = write.QueueCollectorThread(
         event_queue=event_queue,
         writer_dq=writer_dq,
+        writer_queue_length=mp_spawn.Value,
```

Comment:
You are passing the `Value` class here. You probably meant to instantiate a `Value` or use an existing `Value`?